rimage/cse.c

```diff
@@ -1,18 +1,9 @@
-/*
- * Copyright (c) 2017, Intel Corporation.
- *
- * This program is free software; you can redistribute it and/or modify it
- * under the terms and conditions of the GNU General Public License,
- * version 2, as published by the Free Software Foundation.
- *
- * This program is distributed in the hope it will be useful, but WITHOUT
- * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
- * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
- * more details.
- *
- *  Author: Liam Girdwood <liam.r.girdwood@linux.intel.com>
- *          Keyon Jie <yang.jie@linux.intel.com>
- */
+// SPDX-License-Identifier: BSD-3-Clause
```

Comment:
Why are we changing a license? What is the reason why this was GPL in the first place? 
To me the change of license should be a separate PR that's well tracked, not hidden in a script-based code reshuffling.

Reply:
It's not hidden. I explicitly mentioned it in PR description and wrote who needs to agree.

Reply:
well the net result is that we merged a PR that changes the license without explicit agreement from one of the authors.

Reply:
@plbossart original GPL was an error (I probably copy and pasted the wrong licence from somewhere), it should be BSD.

Reply:
@plbossart @lgirdwood Even if it was mistake, the people that was adding code to these files were adding them under GPL license (because in the moment of contribution there was GPL license), so you still need consent from them imho. Just ask @dbaluta for agreement to be safe. He can do it now even if it's already merged, and if he disagrees we can revert this PR.

Reply:
@dbaluta let us know if you disagree with BSD for rimage and we can revert that part.